SDL/macbundle.sh

```diff
@@ -35,11 +35,7 @@ install_name_tool -rpath "${RPATH}" "@executable_path/../Frameworks" "${PPSSPPSD
 
 echo "Done."
 
-GIT_VERSION_LINE=$(grep "PPSSPP_GIT_VERSION = " "$(dirname "${0}")/../git-version.cpp")
-# Hack, need to do something better here.
-if [ -z "$GIT_VERSION_LINE" ]; then
-  GIT_VERSION_LINE=$(grep "PPSSPP_GIT_VERSION = " "$(dirname "${0}")/../build/git-version.cpp")
-fi
+GIT_VERSION_LINE=$(grep "PPSSPP_GIT_VERSION = " "$(dirname "${0}")/../build/git-version.cpp")
```

Comment:
Well, thnis assumes the build directory is what's used?  Probably we would need cmake to pass the path in, right?

-[Unknown]